src/main/java/org/myrobotlab/service/abstracts/AbstractMotor.java

```diff
@@ -417,4 +385,36 @@ public boolean isAttached(String name) {
     return ret;
   }
 
+  public Double getMinX() {
```

Comment:
all of these should delegate the return value from the underlying mapper object.